File: game-server/src/main/java/org/forfun/mmorpg/game/player/message/ReqCreateNewPlayer.java
package org.forfun.mmorpg.game.player.message;

import jforgame.socket.share.annotation.MessageMeta;
import jforgame.socket.share.message.Message;
import org.forfun.mmorpg.game.player.service.PlayerService;

@MessageMeta(cmd = PlayerService.CMD_REQ_CREATE_NEW)
public class ReqCreateNewPlayer implements Message {

	private String name;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

}

File: game-server/src/main/java/org/forfun/mmorpg/game/player/message/ReqCreateNewPlayer.kt
package org.forfun.mmorpg.game.player.message

import jforgame.socket.share.annotation.MessageMeta
import jforgame.socket.share.message.Message
import org.forfun.mmorpg.game.player.service.PlayerService

@MessageMeta(cmd = PlayerService.CMD_REQ_CREATE_NEW)
class ReqCreateNewPlayer : Message {

    var name: String? = null
}
